local/test-binary

```diff
@@ -180,6 +180,25 @@ if [[ "${CI_FS_MTR}" = 'yes' ]]; then
         --junit-package="${DOCKER_OS}.${CMAKE_BUILD_TYPE}.ci_fs" || true
 fi
 
+if [[ "${WITH_PS_PROTOCOL}" = 'yes' ]]; then
+    PS_TESTS=""
+    for PS_TESTS_TMP in $(grep --exclude="*.inc" --exclude="*.log" -rl . -e include/have_ps_protocol.inc | awk -F '/' '{print $NF}' | sed 's/.test//g'); do
```

Comment:
I checked `release-5.7.44-48` and there are no files with `include/have_ps_protocol.inc`.

Reply:
@inikep Do I revert the changes  for 5.7 then?

Reply:
Just close this PR without merging.